### ISSUE TITLE
VS won't compile

### ISSUE THREAD

**Issue by madmax911** [coder-mike/microvium#30]
Hi,

Just wanted to give your program a look but when I went to open the native-vm-vs-project Visual Studio solution it had many errors about missing .h files and many others.  I'll attach a screenshot.

![image](https://user-images.githubusercontent.com/11963048/175658481-02f3b382-6d60-4165-9313-b26b21328cb3.png)

**Comment by coder-mike** [comment]
Did you have a look at the getting-started guide ([doc/getting-started.md](https://github.com/coder-mike/microvium/blob/main/doc/getting-started.md))? The project you're trying run here is not an example of how to use Microvium, it's project used to develop and test Microvium.

Having said that, I'm able to reproduce what you're seeing if I use a clean clone, so something is not quite right. I'll look into it.

**Comment by coder-mike** [comment]
Ok, I've made some changes to fix the compile errors. I think VS was somehow storing some additional state in the `.vs` folder which is not checked into the repo, since if I deleted that folder then it loses the project settings (in this case the include paths and C++ version)

But I still recommend going to [doc/getting-started.md](https://github.com/coder-mike/microvium/blob/main/doc/getting-started.md) if you're just trying it out.

Maybe at some point I need an "examples" folder with pre-baked project files for a couple of different cases.
